node/core/approval-voting/src/persisted_entries.rs

```diff
@@ -76,13 +76,13 @@ impl From<TrancheEntry> for crate::approval_db::v1::TrancheEntry {
 /// particular block.
 #[derive(Debug, Clone, PartialEq)]
 pub struct ApprovalEntry {
-	tranches: Vec<TrancheEntry>,
-	backing_group: GroupIndex,
-	our_assignment: Option<OurAssignment>,
-	our_approval_sig: Option<ValidatorSignature>,
+	pub tranches: Vec<TrancheEntry>,
```

Comment:
I don't like any of this stuff being `pub`. What happens when you remove it that is causing issues?

Reply:
This was due to the code move because the struct is created explicitly in tests. I added a constructor helper and pushed. Should be good to ship